BDArmory/CounterMeasure/CMFlare.cs

```diff
@@ -67,46 +67,49 @@ void OnEnable()
         {
             startThermal = thermal;
             minThermal = startThermal * 0.34f; // 0.3 is original value, but doesn't work well for Tigers, 0.4f gives decent performance for Tigers, 0.65 decay gives best flare performance overall based on some monte carlo analysis
-
-            if (gaplessEmitters == null || pEmitters == null)
+            if (pEmitters == null)
             {
-                gaplessEmitters = new List<BDAGaplessParticleEmitter>();
+                //gaplessEmitters = new List<BDAGaplessParticleEmitter>();
 
                 pEmitters = new List<KSPParticleEmitter>();
 
                 using (var pe = gameObject.GetComponentsInChildren<KSPParticleEmitter>().Cast<KSPParticleEmitter>().GetEnumerator())
                     while (pe.MoveNext())
                     {
                         if (pe.Current == null) continue;
-                        if (pe.Current.useWorldSpace)
-                        {
-                            BDAGaplessParticleEmitter gpe = pe.Current.gameObject.AddComponent<BDAGaplessParticleEmitter>();
-                            gaplessEmitters.Add(gpe);
-                            if (!BDArmorySettings.FLARE_SMOKE && gpe.name != "pEmitter")
-                                gpe.emit = true;
-                        }
-                        else
+                        //if (pe.Current.useWorldSpace)
+                        //{
+                        //    BDAGaplessParticleEmitter gpe = pe.Current.gameObject.AddComponent<BDAGaplessParticleEmitter>();
+                        //    gaplessEmitters.Add(gpe);
+                        //    gpe.emit = true;
+                        //}
+                        //else
```

Comment:
Since there's now the Smoke FX toggle, I guess this could be re-enabled, but while I was able to optimize the particle emitter in the Flare model, GPE is still going to be a performance hit